lightstep/resource_metric_condition_test.go

```diff
@@ -131,15 +114,15 @@ resource "lightstep_metric_condition" "test" {
 	  ]
   }
 }
-`
+`, uqlQuery)
 
-	updatedConditionConfig := `
+	updatedConditionConfig := fmt.Sprintf(`
 resource "lightstep_slack_destination" "slack" {
   project_name = "terraform-provider-tests"
   channel = "#emergency-room"
 }
 
-resource "lightstep_metric_condition" "test" {
+resource "lightstep_alert" "test" {
```

Comment:
does this belong in https://github.com/lightstep/terraform-provider-lightstep/blob/main/lightstep/resource_alert_test.go now?